src/strategies/FluentBiasedRandomStrategy.ts

```diff
@@ -0,0 +1,29 @@
+import {Arbitrary} from '../arbitraries'
+import {FluentResult} from '../FluentCheck'
+import {FluentRandomStrategy} from './FluentRandomStrategy'
+
+export class FluentBiasedRandomStrategy extends FluentRandomStrategy {
```

Comment:
I would prefer composition over inheritance here. For example, a FluentBiasedDecorator(FluentRandomStrategy) would ask the base strategy for inputs and add its own biased values.

Reply:
I thought about that because at first I even thought of making the shrinking process optional.  However, since I agreed with what @ruippeixotog said in #18 I saw fit only to consider the possibility of having bias or not, and therefore inheritance seemed suitable here.

> I always thought of shrinking as a best-effort, nice-to-have feature of PBT.

Nevertheless, I understand your point of view and therefore I will consider your suggestion and change my implementation.